### PR TITLE
Added selecting target language from map - Translation API

### PR DESCRIPTION
**Pull request by furkan-varol**
Edit: New feature: Selecting the target language of the translation from the map.

Implemented text translation functionality. The user is requested to type something in any language in the appropriate input box. The language of that input may or may not be explicitly selected by the user. If it is not, Google Translate API can detect the language and returns it in the response anyway. The location(country) of the user is obtained from a 3rd party API by passing the IP address of the user and this information is used to determine our target language of the translation.

I tested for various conditions and all tests are passed.

> jest --passWithNoTests

 PASS  pages/api/getStatistics.test.js
 PASS  pages/api/getCountriesWithMaxDeaths.test.js
 PASS  services/getConvertedPrice.spec.js
 PASS  services/getLocationInfo.spec.js
 PASS  pages/api/getRedditData.test.js
 PASS  pages/api/getTranslation.test.js

Test Suites: 6 passed, 6 total
Tests:       45 passed, 45 total
Snapshots:   0 total
Time:        5.107 s
Ran all test suites.

### CODE REVIEW

File: practice_app/pages/index.js [hunk 1]
```diff
@@ -198,14 +199,13 @@ export default function Home({ context }) {
     // current functionality : source_lang=english, target_lang=ip_dependent
     const getTranslation = async () => {
         try {
-            // Learn client's ip adress using a third party service
-            const ipInfo = await axios.get('https://json.geoiplookup.io/')
-            // Provide client's ip & original text to our api
+            const isoConv = require('iso-language-converter')
```
Comment (MehdiSaffar) [inline practice_app/pages/index.js]: require(..) should not be used in client side stuff. Import it at the top of the file like this
```js
import isoConv from 'iso-language-convert' // OR
import * as isoConv from 'iso-language-convert'
```
whichever works.

**Review by MehdiSaffar — CHANGES_REQUESTED** [review]
All the tests pass on my machine, and I tried the feature by changing the location from the map and retranslating. Good job!

Once you fix the import thing, you can go ahead and merge this branch

File: practice_app/pages/index.js (hunk 1 above)
Comment (furkan-varol) [reply practice_app/pages/index.js]: they did not work so i moved the conversion to the server side

**Review by MehdiSaffar — APPROVED** [review]
Excellent job, can be merged

**Pull request merged**